{"sections":[],"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cancelled","kind":"identifier"}],"languages":["swift"],"platforms":["macOS"]}]}],"abstract":[{"type":"text","text":"Awaiting call to connect() was cancelled."}],"variants":[{"paths":["\/documentation\/yubikit\/connectionerror\/cancelled"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError\/cancelled","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/YubiKit\/documentation\/YubiKit","doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError"]]},"metadata":{"symbolKind":"case","title":"ConnectionError.cancelled","roleHeading":"Case","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cancelled","kind":"identifier"}],"role":"symbol","modules":[{"name":"YubiKit"}],"externalID":"s:7YubiKit15ConnectionErrorO9cancelledyA2CmF"},"references":{"documentation-header.png":{"type":"image","identifier":"documentation-header.png","variants":[{"traits":["2x","light"],"url":"\/images\/YubiKit\/documentation-header@2x.png"},{"traits":["2x","dark"],"url":"\/images\/YubiKit\/documentation-header~dark@2x.png"}],"alt":"YubiKeys in a row"},"doc://YubiKit/documentation/YubiKit":{"images":[{"identifier":"documentation-header.png","type":"icon"}],"title":"YubiKit","kind":"symbol","role":"collection","identifier":"doc:\/\/YubiKit\/documentation\/YubiKit","url":"\/documentation\/yubikit","abstract":[{"type":"text","text":"Connect and run commands on the different applications on a YubiKey. The framework support connecting using NFC, Lightning and USB-C."}],"type":"topic"},"doc://YubiKit/documentation/YubiKit/ConnectionError":{"type":"topic","title":"ConnectionError","url":"\/documentation\/yubikit\/connectionerror","identifier":"doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError","abstract":[{"type":"text","text":"SmartCardConnection Errors."}],"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"ConnectionError"}],"kind":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ConnectionError"}]},"doc://YubiKit/documentation/YubiKit/ConnectionError/cancelled":{"title":"ConnectionError.cancelled","abstract":[{"text":"Awaiting call to connect() was cancelled.","type":"text"}],"kind":"symbol","url":"\/documentation\/yubikit\/connectionerror\/cancelled","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cancelled"}],"identifier":"doc:\/\/YubiKit\/documentation\/YubiKit\/ConnectionError\/cancelled","role":"symbol","type":"topic"}}}